apis/cat.py

```diff
@@ -0,0 +1,33 @@
+from flask_restplus import Namespace, Resource, fields
+
+api = Namespace('cats', description='Cats related operations')
+
+cat = api.model('Cat', {
+    'id': fields.String(required=True, description='The cat identifier'),
+    'name': fields.String(required=True, description='The cat name'),
+})
+
+CATS = [
+    {'id': 'felix', 'name': 'Felix'},
+]
+
+@api.route('/')
+class CatList(Resource):
```

Comment:
What's `cat` standing for?

Reply:
It's a sample code from 
https://flask-restplus.readthedocs.io/en/stable/scaling.html.
And I just copied it without any revision.